### ISSUE TITLE
New license request: Sendmail-2009 [SPDX-Online-Tools]

### ISSUE THREAD

**Issue by xsuchy** [spdx/license-list-XML#2565]
**1.** License Name: Sendmail 2009 License
**2.** Short identifier: Sendmail-2009
**3.** License Author or steward: Unknown
**4.** Comments: This license was discovered during Fedora's Linux license review in https://gitlab.com/fedora/legal/fedora-license-data/-/issues/568
**5.** License Request Url: http://tools.spdx.org/app/license_requests/393
**6.** URL(s): https://github.com/trusteddomainproject/OpenDMARC/blob/master/LICENSE.Sendmail
**7.** OSI Status: Unknown
**8.** Example Projects: https://github.com/trusteddomainproject/OpenDMARC

**Comment by xsuchy** [comment]
Text of the license:
```
 SENDMAIL OPEN SOURCE LICENSE

The following license terms and conditions apply to this open source
software ("Software"), unless a different license is obtained directly
from Sendmail, Inc. ("Sendmail") located at 6475 Christie Ave, Suite 350,
Emeryville, CA 94608, USA.

Use, modification and redistribution (including distribution of any
modified or derived work) of the Software in source and binary forms is
permitted only if each of the following conditions of 1-6 are met:

1. Redistributions of the Software qualify as "freeware" or "open
   source software" under one of the following terms:

   (a) Redistributions are made at no charge beyond the reasonable
       cost of materials and delivery; or

   (b) Redistributions are accompanied by a copy of the modified
       Source Code (on an acceptable machine-readable medium) or by an
       irrevocable offer to provide a copy of the modified Source Code
       (on an acceptable machine-readable medium) for up to three years
       at the cost of materials and delivery. Such redistributions must
       allow further use, modification, and redistribution of the Source
       Code under substantially the same terms as this license. For
       the purposes of redistribution "Source Code" means the complete
       human-readable, compilable, linkable, and operational source
       code of the redistributed module(s) including all modifications.

2. Redistributions of the Software Source Code must retain the
   copyright notices as they appear in each Source Code file, these
   license terms and conditions, and the disclaimer/limitation of
   liability set forth in paragraph 6 below. Redistributions of the
   Software Source Code must also comply with the copyright notices
   and/or license terms and conditions imposed by contributors on
   embedded code. The contributors' license terms and conditions
   and/or copyright notices are contained in the Source Code
   distribution.

3. Redistributions of the Software in binary form must reproduce the
   Copyright Notice described below, these license terms and conditions,
   and the disclaimer/limitation of liability set forth in paragraph
   6 below, in the documentation and/or other materials provided with
   the binary distribution.  For the purposes of binary distribution,
   "Copyright Notice" refers to the following language: "Copyright (c)
   1998-2009 Sendmail, Inc. All rights reserved."

4. Neither the name, trademark or logo of Sendmail, Inc. (including
   without limitation its subsidiaries or affiliates) or its contributors
   may be used to endorse or promote products, or software or services
   derived from this Software without specific prior written permission.
   The name "sendmail" is a registered trademark and service mark of
   Sendmail, Inc.

5. We reserve the right to cancel this license if you do not comply with 
   the terms.  This license is governed by California law and both of us 
   agree that for any dispute arising out of or relating to this Software, 
   that jurisdiction and venue is proper in San Francisco or Alameda 
   counties.  These license terms and conditions reflect the complete 
   agreement for the license of the Software (which means this supercedes 
   prior or contemporaneous agreements or representations).  If any term
   or condition under this license is found to be invalid, the remaining
   terms and conditions still apply.

6. Disclaimer/Limitation of Liability: THIS SOFTWARE IS PROVIDED BY
   SENDMAIL AND ITS CONTRIBUTORS "AS IS" WITHOUT WARRANTY OF ANY KIND
   AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
   IMPLIED WARRANTIES OF MERCHANTABILITY, NON-INFRINGEMENT AND FITNESS FOR A
   PARTICULAR PURPOSE ARE EXPRESSLY DISCLAIMED. IN NO EVENT SHALL SENDMAIL
   OR ITS CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
   SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
   TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA,
   OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
   OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
   WITHOUT LIMITATION NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
   USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.

$Revision: 1.1 $ $Date: 2009/07/16 18:43:18 $
```

Note that @richardfontana has already done some investigation in https://gitlab.com/fedora/legal/fedora-license-data/-/issues/568#note_2117606078 that is worth reading.

**Comment by swinslow** [comment]
So I had written a comment preparing to recommend adding this with the ID `Sendmail-1.1`, noting the revision number on the last line. But after doing some more initial digging, I think it might be worth doing a bit more research on this one.

I'm not familiar with the history of Sendmail, but it looks to me like there might be two separate licenses (closely related, but separate lineage):

1. The "Sendmail License", which is what we currently have on the list as [`Sendmail`](https://spdx.org/licenses/Sendmail.html) (dated 2010, revision number 8.16) and [`Sendmail-8.23`](https://spdx.org/licenses/Sendmail-8.23.html) (dated 2014, revision number 8.23); and
2. The "Sendmail Open Source License", which is in the version submitted here (dated 2009, revision number 1.1); I've also found an earlier version (dated 2007, also with revision number 1.1) at https://ftp.netbsd.org/pub/pkgsrc/current/pkgsrc/licenses/sendmail-open-source-license, identical to the 2009 version other than changes in addresses / dates.

The reason I think these are separate "threads" of licenses, rather than the 2007 / 2009 just being an earlier version of those on the list currently, is that I've also found a 2003 version of the "Sendmail License" at: https://athena10.mit.edu/trac/browser/trunk/third/sendmail/LICENSE?rev=21373&desc=1. This is nearly identical to the versions on the list now (those in item 1 above); but it pre-dates the "Sendmail Open Source License" versions in item 2 above, suggesting that those were separately developed in parallel.

The reason I'm going into this level of detail is that I think the version submitted in this issue is perhaps more properly framed as the "Sendmail Open Source License", perhaps with an ID like `Sendmail-Open-Source-1.1`, rather than just bundling it in as another `Sendmail-X.y` license. It would be great if someone who knows the history with Sendmail's licensing could weigh in rather than me just guessing.  :)

**Comment by swinslow** [comment]
Note also https://www.sendmail.org/~ca/email/sm-X/, from 2006, which indicates: "Sendmail X had a slightly different LICENSE than sendmail 8 until 2006-05-19." This may help point to why these separate licenses exist, though I don't have more time at this moment to dig deeper unfortunately...

**Comment by Pizza-Ria** [comment]
Some of the changes seem "clarifying" or just more succinct than the original language but there is enough that has changed substantively that I vote for giving this an version bump on the 8.16 version of the SendMail license.  For instance, this language "For the purposes of redistribution "Source Code" means the complete human-readable, compilable and, linkable source code of sendmail, and operational source code of the redistributed module(s) including all modifications" refines the parameters of what "source code" means.  Plus a choice of law was added (additional restriction?).

**Comment by jlovejoy** [comment]
also would be good to add clarifying notes on existing Sendmail variants and this one

**Comment by swinslow** [comment]
# License Inclusion Decision

## Decision:

- [X] approved
- [ ] not approved

## Name

Sendmail Open Source License v1.1

## License ID

`Sendmail-Open-Source-1.1`

## XML markup

Similar to markup used for [`Sendmail`](https://spdx.org/licenses/Sendmail.html)

## Notes:



## Next steps ##
If the license has been accepted, please follow [the accepted-license process to create the PR](https://github.com/spdx/license-list-XML/blob/main/DOCS/request-new-license.md#accepted-license-process).
